ci/docker/base.test.Dockerfile

```diff
@@ -60,3 +60,4 @@ WORKDIR /ray
 COPY . .
 
 RUN ./ci/env/install-bazel.sh
+
```

Comment:
why? maybe just revert this change? (and I do not need to review this PR.)

Reply:
it was for some cache-related issue test, will remove!!
cc: @can-anyscale